index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+   <link rel="stylesheet" href="style.css">
+  <title>Personal Portfolio</title>
+</head>
+<body>
+   <header>
+      <nav>
+        <div class="logo"><a href="#">I<span>.J</span></a></div>
+         <div class="hamburger-menu"><i class="fa-solid fa-bars"></i></div>
+      </nav>
+   </header>
+
+   <section class="title">
+     <h1>I'm Isaac. Glad To see you!</h1>
+
+     <p>
+      I'm a Software developer! Can help you build a product, feature or Website Look through Some of my Work and experience! if you like What you see and have a project you need coded, don't hesitate to contact me.
+     </p>
+
+    <span class="span">LET'S CONNECT</span>
+    <div class="socials">
+        <a href="https://twitter.com/ejama_isaac"> <img src="img/Vector (3).svg" alt=""> </a>
+        <a href=""><img src="img/Linkedin icon.svg" alt=""></a>
+        <a href="#"><img src="img/Vector (2).svg" alt=""></a>
+        <a href="https://github.com/isaac-ej"><img src="img/Vector (1).svg" alt=""></a>
+        <a href="#"><img src="img/Vector.svg" alt=""></a>
+    </div>
```

Comment:
- [ ] It is important to make use of semantic tags as much as possible in our application and as such i will suggest rather than using the non-semantic `div` tag as container for the social media icons, you make use of the semantic `ul` tag and each link would be wrapped in an `li` tag as a list item. see the example below for more reference :+1: 
```html
<ul class="socials">
  <li>
    <a href="https://github.com/"><img src="./assets/github.png" alt="github"/></a>
  </li>
  <li>
    <a href="https://linkedin.com/in/"><img src="./assets/linkedin.png" alt="linkedin"/></a>
  </li>
</ul>
```